lib/reporters/dependencies/audit.js

```diff
@@ -0,0 +1,39 @@
+const { getLockFilesExistence} = require('../../util');
+const { execSync } = require( 'child_process');
+
+function generateYarnAuditReport(projectRootPath) {
+	const originalCwd = process.cwd();
+
+	try {
+		execSync(`yarn audit --cwd "${projectRootPath}" --json | ${originalCwd}/node_modules/.bin/yarn-audit-html`);
+	} catch (err) {
+		console.log('generateYarnAuditReport error: ', err);
+	}
+}
+
+function generateNpmAuditReport(projectRootPath) {
+	const originalCwd = process.cwd();
+	process.chdir(projectRootPath);
+
+	try {
+		execSync(`npm audit --json | ${originalCwd}/node_modules/.bin/npm-audit-html`);
+	} catch (err) {
+		console.log('generateNpmAuditReport error: ', err);
+	} finally {
+		process.chdir(originalCwd);
+	}
+}
+
+const performAudit = ({ projectRoot }) => {
+	const { packageLockExist, yarnLockExist } = getLockFilesExistence(projectRoot);
+
+  	if (yarnLockExist) {
+		generateYarnAuditReport(projectRoot);
+	} else if (packageLockExist) {
+		generateNpmAuditReport(projectRoot);
+	} else {
+		// pnpm support goes here
```

Comment:
@arielj should the tool raise an error here? that way it won't fail silently if project is using `pnpm`

Reply:
if we raise an error here it will make the other tools crash, maybe I can print something in the console instead

Reply:
I updated the code to print in the console that these are not supported for pnpm yet